nuv/kind.go

```diff
@@ -18,6 +18,7 @@
 package main
```

Comment:
The basic idea is:

if <I am in a container>
  if DOCKER_HOST is set
     exit
  else
      sudo env DOCKER_HOST=.... + os.Args()